### PR TITLE
add minimal docs + collect telemetry while running the codemod

### LINKED ISSUE #28: Codemod is adding `this.` to *everything*
I ran `npx ember-no-implicit-this-codemod http://localhost:4200 app/` and this is what happened...

Bildschirmfoto 2019-10-04 um 12 27 48

to reproduce, https://github.com/skylines-project/skylines/tree/4af6a8173ccddd5339f79c2a44a97615b682231c/ember is the commit that I ran this on

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Resolves #28 and #27

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/env node
 'use strict';
 
-const transformName = process.argv[2];
+const appLocation = process.argv[2]
```
Comment (rwjblue) [inline bin/cli.js]:
```suggestion
const appLocation = process.argv[2];
```

**Comment by NullVoxPopuli** [comment]
There are some interop issues with newer ember :-\
so, this codemod may need to be ran pre 3.13?

**Comment by rwjblue** [comment]
Not sure I understand, can you open an issue to track?

**Comment by NullVoxPopuli** [comment]
There's an in progress PR here: https://github.com/ember-codemods/ember-codemods-telemetry-helpers/pull/12

**Pull request merged**